include/vcpkg/base/jsonreader.h

```diff
@@ -17,21 +17,21 @@ namespace vcpkg::Json
     struct IDeserializer
     {
         using type = Type;
-        virtual StringView type_name() const = 0;
+        virtual LocalizedString type_name() const = 0;
```

Comment:
Are you sure to change StringView to LocalizedString? I think change StringView to LocalizedStringView